tests/unit/test_BufferedSingerStream.py

```diff
@@ -75,6 +76,73 @@ def test_add_record_message__invalid_record():
     assert [] == missing_sdc_properties(singer_stream)
 
 
+SIMPLE_MULTIPLE_OF_VALID_SCHEMA = {
+    'properties': {
+        'multipleOfKey': {
+            'type': 'number',
+            'multipleOf': Decimal('1e-15')
+        }
+    }
+}
+
+SIMPLE_MULTIPLE_OF_INVALID_SCHEMA = {
+    'properties': {
+        'multipleOfKey': {
+            'type': 'number',
+            'multipleOf': 1e-15
+        }
+    }
+}
+
+def test_add_record_message__multipleOf():
+    stream_name = 'test'
+    singer_stream = BufferedSingerStream(stream_name,
+                                         deepcopy(SIMPLE_MULTIPLE_OF_VALID_SCHEMA),
+                                         [])
+
+    multiple_of_values = ['1', '2', '3', '4', '5', '1.1', '2.3', '1.23456789', '20', '100.1']
+
+    for value in multiple_of_values:
+        singer_stream.add_record_message(
+            {
+                'type': 'RECORD',
+                'stream': stream_name,
+                'record': {'multipleOfKey': Decimal(value)},
+                'sequence': 0,
+                RAW_LINE_SIZE: 100
```

Comment:
I think we can drop this. It's not really adding value to the test overall.

Reply:
actually, we need to keep this. Because:

1. you add it here: https://github.com/datamill-co/target-postgres/blob/master/target_postgres/target_tools.py#L139
2. and then, when you call `get_line_size` here https://github.com/datamill-co/target-postgres/blob/master/target_postgres/singer_stream.py#L152
3. we return `line_data.get(RAW_LINE_SIZE)` (doing `json.dumps(line_data)` crashes, because of the Decimal)

Reply:
Ah, damnit, sorry, fat fingers. I meant to mark the _similar line_ as the other comment I left. You're absolutely _right_ that _this_ line is necessary.